plover/steno_dictionary.py

```diff
@@ -264,7 +279,7 @@ def save(self, path_list=None):
 
         If <path_list> is None, all writable dictionaries are saved'''
         if path_list is None:
-            dict_list = [d for d in self if dictionary.save is not None]
+            dict_list = [d for d in self if not d.readonly]
```

Comment:
Why?

Reply:
It checks for the existence of a save method rather than the readonly property. Even if a dictionary has this method we may not want it saved if readonly is set.

(also my IDE threw up an error over the name "dictionary", I don't see it defined in that namespace.)